http/parse.go

```diff
@@ -162,17 +168,16 @@ func parseRequest(r *http.Request, root *cmds.Command) (*cmds.Request, error) {
 	return req, err
 }
 
-func parseOptions(r *http.Request) (map[string]interface{}, []string) {
-	opts := make(map[string]interface{})
+func parseOptions(r *http.Request) (map[string][]string, []string) {
```

Comment:
I'd consider removing this helper. Up to you but it's looking like that might actually simplify the code at this point.

Reply:
done